clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -304,6 +307,11 @@ class ServerGroupController {
           instanceType = serverGroup.launchConfig.instanceType
         }
       }
+
+      if (serverGroup.launchTemplate && serverGroup.launchTemplate.launchTemplateData) {
+        instanceType = serverGroup.launchTemplate.launchTemplateData.instanceType
```

Comment:
Slightly more complicated.

You could also do a `objectMapper.convertValue(serverGroup, Map)` here and just reference attributes out of a map. 

That would at least be consistent across providers.